items.json

```diff
@@ -10400,18 +10400,18 @@
         "missileSpeed": 0,
         "stackAmount": 0,
         "length": 100,
-        "balance": 0.51,
-        "handling": 89,
+        "balance": 0.61,
+        "handling": 91,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon"
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
-        "thrustSpeed": 90,
-        "swingDamage": 15,
+        "thrustSpeed": 92,
+        "swingDamage": 23,
         "swingDamageType": "Blunt",
-        "swingSpeed": 85
+        "swingSpeed": 88

```

Comment:
can you go for lower swing speed higher damage?

Reply:
made it slower and stronger